src/main/resources/db/migration/V0001__Base_version.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE public.case_payment_orders (
+                         id uuid not null PRIMARY KEY,
+                         created_timestamp timestamp without time zone not null,
```

Comment:
I might be wrong but I think `timestamp without time zone` is for local dates, whereas we need to store dates with time zones. I need to do a bit more research since this is a complex subject